cmd/gator/commons/commons.go

```diff
@@ -0,0 +1,22 @@
+package commons
+
+import (
+	"fmt"
+	"os"
+)
+
+func ErrFatalf(format string, a ...interface{}) {
```

Comment:
good refactor. `cmd/gator/expand/expand.go` should use this helper too.

I'd suggest the change but I don't think I can on a file that hasn't been modified in the PR yet.

Reply:
That's a good callout! I was going to kick down whole sale usage of this to a separate PR but can add it in this one to reduce review roundtrips.